elearning-frontend/src/components/Login.js

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+import apiClient from '../services/api';
+import { useNavigate } from 'react-router-dom';
+
+const Login = (props) => {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const navigate = useNavigate();
+
+    useEffect(() => {
+        if (props.session) {
+            navigate('/dashboard')
+        }
+    }, [navigate, props.session])
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        localStorage.removeItem('user');
+
+        apiClient.get('/sanctum/csrf-cookie').then(response => {
+            apiClient({
+                method: "post",
+                url: "/api/login",
+                data: {
+                    email: email,
+                    password: password
+                }
+            }).then((response) => {
+                localStorage.setItem('user', response.data.token)
+                props.login();
+                navigate('/dashboard')
+            }).catch(error => {
+                console.log(error);
+            });
+        }).then(response => {
+            console.log(response);
+        }).catch(error => {
+            console.log(error);
+        });
+    }
```

Comment:
```suggestion
    const handleSubmit = async (e) => {
        e.preventDefault();
        localStorage.removeItem('user');

        const checklogin = await apiClient.get('/sanctum/csrf-cookie')
        
         if (checkLogin === 'some condition here') {
         apiClient({
                method: "post",
                url: "/api/login",
                data: {
                    email: email,
                    password: password
                }
            }).then((response) => {
                localStorage.setItem('user', response.data.token)
                props.login();
                navigate('/dashboard')
            }).catch(error => {
                console.log(error);
            });
         } else{
         some error here
         }
    }
```

Dont put api call inside api call